src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -51,6 +51,12 @@ public sealed partial class FileSystemProvider : NavigationCmdletProvider,
                                                      ISecurityDescriptorCmdletProvider,
                                                      ICmdletProviderSupportsHelp
     {
+#if UNIX
+        // This is the errno returned by the rename() syscall
+        // when an item is attempted to be renamed across filesystem mount boundaries.
+        private const int UNIX_ERRNO_EXDEV = 18;
```

Comment:
18 appears to be correct for Linux, macOS, and FreeBSD.

Reply:
We should check the error code is 18 for all the variants we support especially Raspbian and Alpine, which could have it a bit different.

Reply:
Confirmed on RPi:

```
pi@daedalus:~ $ errno -l | grep EXDEV
EXDEV 18 Invalid cross-device link
```

Reply:
Confirmed on Alpine:

```
/ # errno -l | grep EXDEV
EXDEV 18 Cross-device link
```